assets/src/editor/events/dates-and-tickets-metabox/dates-and-tickets-metabox.js

```diff
@@ -96,13 +80,8 @@ const MetaBox = ( {
 							className="ee-editor-event-tickets ee-form-row">
 							<div>
 								<EditorTicketEntitiesList
-									loading={ ! ticketEntitiesLoaded }
-									entities={ filteredTicketEntities }
-									allDateEntities={ allDateEntities }
-									isChained={ isChained }
-									prefiltered
+									loading={ ! initialized }
```

Comment:
I realize this is a change in that now both entity lists will show the loading indicator until all dates and tickets are initialized from the server.  In a future iteration we can look at making the initialized context value more granular so we can stagger the loading.  However, I want to hold off doing that until I introduce the hydration mechanism to the initialization layer to see whether we need the granular initialization state exposed.

Reply:
Right, I expected those improvements because there's in editor-dates-tickets-refactor the three hocs now in the initialization layer were repeatedly getting triggered and thus causing re-renders in the component tree because of the change in props.  Thanks for posting this though, it's nice to get some hard date to confirm it helped things 👍

Reply:
oh and just for clarity, those numbers I posted only apply to the one particular event I have been using for development purposes, so those numbers are a reflection of the number of dates, tickets, and relations for that event, and would not be the same for any other event.

That said, though this branch does appear to be a good 40% faster which is a very significant improvement, so ya, good work!